### DISCUSSION TITLE
Scale the network coordinates to export to Paraview

### DISCUSSION THREAD

**Discussion by engphb** [PMEAL/OpenPNM#2613] [Q&A] [answered]
Hi professor, how can I scale the network coordinates and pore and throat diameters to the image dimensions before exporting to Paraview? My network after the snow algorithm has meter dimensions, I need scaling to micrometer dimensions. Below is the main part of my code where I export the network to .vtk. Another question: Is it possible to export the created network to .tiff?

path ='/content/poros.tiff'
file_name = 'poros'
file_format = '.tif'
file = file_name + file_format
image = os.path.join(path, file)
fetch_file = os.path.join(path, file)
im = imageio.mimread('/content/poros.tiff', memtest=False)
im = np.array(im, dtype=bool)

resolution = 12.0550000548155e-6 
net=ps.networks.snow2(im, voxel_size=resolution, accuracy='standard', sigma=0.4, r_max=4)

ws = op.Workspace()
proj = op.Project()
pn = op.network.GenericNetwork(name=file_name, project=proj)
pn.update(net.network) # Fills 'pn' with data from 'net'

op.io.VTK.save(network=pn, filename='network')

**Comment by jgostick** [comment, marked as answer]
Hi @engphb 
I think you emailed me about this right?  This question feels very familiar.

**Reply by engphb** [reply]
Yes professor. You already answered me by email. Thank you!
